compose/service.py

```diff
@@ -679,7 +679,8 @@ def labels(self, one_off=False):
 
     def can_be_scaled(self):
         for port in self.options.get('ports', []):
-            if ':' in str(port):
+            external = split_port(port)[1]
+            if external and not (type(external) == tuple and external[1] == None):
```

Comment:
Oh, and [PEP 8](https://www.python.org/dev/peps/pep-0008/) says to use `is None`, not `== None`.

> Comparisons to singletons like None should always be done with is or is not , never the equality operators